### PR TITLE
Added new notebook with DIA Run2 introduction

### PR DESCRIPTION
**Pull request by BrunoSanchez**
Adding notebook for mini sprint session with DIA

### CODE REVIEW

**Comment by wmwv** [comment]
@BrunoSanchez The image doesn't come through for me.  Please add `processing_schema.png` to the tutorials directory.

**Comment by BrunoSanchez** [comment]
Sorry, now this should be OK.

**Comment by wmwv** [comment]
I think it's really important to add a note about how to screen the diaSrc for the sources that weren't flagged as obviously bad: e.g., SATURATED, INTERPOLATED.  Looking at a raw diaSrc catalog without removing things already flagged to not be good creates a very confusing impression of the purity of the subtractions and detections.

File: contributed/DIA_Run2_intro.ipynb [hunk 1]
```diff
@@ -0,0 +1,2123 @@
...
...SFkmMvXY6VZEvn5f2PIbP8qBLH1qJHU3YiLIPx2X4ag+oVI3bzsFZH7QrvJwxVEUJmSDz+cb9o0axLutYLYPmUFUu6tKFb0vKyS8tZ/RT/2Rc765MSXKQHhtNVnEpHsOoDIrQeGzyeCLsjXdr/XXpCv67fiNe01+j8ZPvt/ODgf357cUTQ3iGjadnUiJLH7ib+bv3crysnMFpHRjduVNQhivcbqNLXCw2Ta1eF5XAK6vWkBQRzvUD+wfdj8WHD3L/17MRCExp8eeVS7lr0DAeHdP2CsDaKwOgAm5TWu9ROCx+HIvyPq+sql6b/jHB5QI+OngCK48fwm3o6NJCEQKHovKnUVPrXROzj72JyyyrjnQ1pA9D+riidz5z96RQ4vUiBOiWxS+HXMTIlMatqZ4ttBu/NkiFz0eBy9WsfKNziZHp6Tx76SX8YdHi6qCXS3v2xJ3n5XhxGZaUlHl8fL11Dz0GduTNm6/hm937Wbh7H4mREdw6fBBD0huvVn+wsIg312+sZXTdusGHK7eyZsVBCkpcdEmJ4+fTL2R05pmvth5ut3H1gMZLcRmWxf82b6k2fFW4DYN/rV4btPFz6ToPfP0l7jqz7De3bmRi566MSKuXotuqZETF0ikqln3FJ2rdW2GajVsyW28JIt6ezA2d7uODo/+qpSxze5eHcarBxQR0jY7n6yvu5vUda9lwIotu0Qnc22cUmXH117h3l22pl+IBUGbtZOnVv2XLieOU+LwMS+pIjOPcFeFoN35tCN00eXrxYj7+fjuqonBfhxT2r1/P3cPPWOpLm+XKzEym9epFbnk50U4nx4vK+MHf36s1krekxJKSeZt3c/tFw7ltxOBmHXPR/gP1FEW0ClCKLLKkP9py59E8Zr46m+fvmc7YPmfeADYFj24ETBYHf8BQsCw9eqhWsFF1+4bBJ7t3tDnjB/DvSTO4Yd77uA0d0/I7CSd36s4NPYNXh2kJBseNJTN6CPvLt6MIlR6R/bApjfNUpIZH89vhpw+w14SGIevX81OEiiIUhiW3vd+tJWg3fm2Ivyxdxifbd/gDEUwTS0peWLGS5IhIpvfJbO3utTqqopAWHQ3AnuMn0NST+VNSSCyb3wBuPnK80W3nu5ZxtHQWhiynQ/hUOkZdjVPT/MVuq1bGJNhLQNQZNHt0gxe/WHbWGL8Iu43EiHCOl9WvYtC/Q0r
```
Comment (wmwv) [inline contributed/DIA_Run2_intro.ipynb]: Write a function that does this.  Call it something informative.
Then call the function from the loop.

Comment (wmwv) [inline contributed/DIA_Run2_intro.ipynb]: Write a function that does this.  Call it something informative.
Then call the function from the loop.

Comment (wmwv) [inline contributed/DIA_Run2_intro.ipynb]: Just `lightcurve`.  User can control output.

Comment (wmwv) [inline contributed/DIA_Run2_intro.ipynb]: Add some more text explaining the steps you're going to do below.  Something like

```
1. Load DIAObject table and select an object
2. Get the diaSrc IDs from the DIAObject table for that object
3. Use a magic trick to get the dataIds that allow us to load the diaSrc catalogs for each observation of the source.  These are the measurements from the detections on each image.  They are _not_ the forced photometry on all available images.
4. Load diaSrc values for each detection.
5. Calibrate the diaSrc instFlux measurements
6. Plot lightcurve
```

Comment (wmwv) [inline contributed/DIA_Run2_intro.ipynb]: Link to postage stamp Notebook.
```
We could also use the techniques in [this Notebook](dia_source_object_stamp.ipynb) to create stamps, but for the moment this is enough information!
```

Comment (BrunoSanchez) [reply contributed/DIA_Run2_intro.ipynb]: Thank you for this, it really improves the introduction part.

Comment (BrunoSanchez) [reply contributed/DIA_Run2_intro.ipynb]: Will do

[the remark this answers is not in the document]
Comment (BrunoSanchez) [reply contributed/DIA_Run2_intro.ipynb]: Done

File: contributed/DIA_Run2_intro.ipynb [hunk 2]
```diff
@@ -0,0 +1,2123 @@
...
+       "[5 rows x 248 columns]"
+      ]
+     },
+     "execution_count": 19,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "srcs.head()"
```
[the remark this answers is not in the document]
Comment (BrunoSanchez) [reply contributed/DIA_Run2_intro.ipynb]: Done

File: contributed/DIA_Run2_intro.ipynb [hunk 3]
```diff
@@ -0,0 +1,2123 @@
...
+      "diaSrc_00159479-g-R03-S02-det020.fits  diffexp_00159479-g-R03-S02-det020.fits\n",
+      "diaSrc_00159479-g-R03-S11-det022.fits  diffexp_00159479-g-R03-S11-det022.fits\n",
+      "diaSrc_00159479-g-R03-S12-det023.fits  diffexp_00159479-g-R03-S12-det023.fits\n",
+      "diaSrc_00159479-g-R03-S21-det025.fits  diffexp_00159479-g-R03-S21-det025.fits\n",
+      "diaSrc_00159479-g-R03-S22-det026.fits  diffexp_00159479-g-R03-S22-det026.fits\n"
+     ]
+    }
+   ],
+   "source": [
+    "ls /global/cscratch1/sd/bos0109/templates_rect/rerun/diff_rect/deepDiff/v00159479-fg/R03"
```
[the remark this answers is not in the document]
Comment (BrunoSanchez) [reply contributed/DIA_Run2_intro.ipynb]: Great suggestion, I have implemented this. I think this is the cleanest example I have seen for doing this.

[the remark this answers is not in the document]
Comment (BrunoSanchez) [reply contributed/DIA_Run2_intro.ipynb]: I followed this suggestion. Please confirm that I did this correctly

File: contributed/DIA_Run2_intro.ipynb (hunk 1 above)
[the remark this answers is not in the document]
Comment (BrunoSanchez) [reply contributed/DIA_Run2_intro.ipynb]: Thanks, done.

[the remark this answers is not in the document]
Comment (BrunoSanchez) [reply contributed/DIA_Run2_intro.ipynb]: Thanks for this, I always forget this.

[the remark this answers is not in the document]
Comment (BrunoSanchez) [reply contributed/DIA_Run2_intro.ipynb]: Fixed.

Comment (BrunoSanchez) [reply contributed/DIA_Run2_intro.ipynb]: I don't know what particular function you have in mind. Will ask you in person later.

**Comment by BrunoSanchez** [comment]
Check it now. I think most of the comments are resolved. Last comment regarding function uses, would like to see which specific parts should be functions, so I can make them informative.

File: contributed/DIA_Run2_intro.ipynb (hunk 1 above)
Comment (wmwv) [reply contributed/DIA_Run2_intro.ipynb]: My general philosophy is that the body of a loop should be a function.  That allows one to (a) test the function without running the loop and (b) take the opportunity to provide an informative name for the function to help document what the loop does.

**Comment by reneehlozek** [comment]
I think this is great @BrunoSanchez  - and I'm sure there'll be more comments after the mini sprint today

**Comment by reneehlozek** [comment]
hi @BrunoSanchez and @wmwv, are the things in place to make these changes/updates given the recent progress?

**Comment by BrunoSanchez** [comment]
I am not sure. I think I have resolved everything requested. 
Michael still needs to approve on his side I think.

**Comment by yymao** [comment]
@BrunoSanchez I think we can merge this. Can you first clear the output of the notebook? See Step 5 of [this instructions](https://github.com/LSSTDESC/DC2-analysis/tree/master/contributed#contribute-your-analysis-notebookscode).

**Pull request merged**